deluge/metafile.py

```diff
@@ -10,10 +10,14 @@
 # See LICENSE for more details.
 #
 
+import copy
 import logging
 import os.path
 import time
 from hashlib import sha1 as sha
+from hashlib import sha256
+
+import libtorrent as lt
```

Comment:
Cannot import libtorrent here, this burdens UI clients with libtorrent requirement

Reply:
I imported libtorrent in the corresponding function only